src/anim/binder/default-anim-binder.js

```diff
@@ -156,6 +156,10 @@ class DefaultAnimBinder {
         let node;
         if (this.graph) {
             node = this.graph.findByPath(path.entityPath);
+            // if the path is not found under the given root node, try to find the path using the root node as the base of the path instead
+            if (!node) {
+                node = this.graph.findByPath(path.entityPath.slice(1));
```

Comment:
shouldn't this be `slice(1, 1)` ?

Reply:
oh, no, ignore me :)

Reply:
from the PR description it looks like it's the second child that needs removing, not the root?

Reply:
path.entityPath is an array of strings (each string being a node in the path). We want to search using index 1 to the end of the path.